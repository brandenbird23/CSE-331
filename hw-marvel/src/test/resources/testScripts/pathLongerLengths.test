# Tests to find the path between nodes when it is longer than 1

LoadGraph nfl nflPlayers.csv

FindPath nfl RUSSEL-WILSON TOM-BRADY