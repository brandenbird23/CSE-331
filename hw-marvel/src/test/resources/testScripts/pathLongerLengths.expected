# Tests to find the path between nodes when it is longer than 1]

loaded graph nfl

path from RUSSEL-WILSON to TOM-BRADY:
RUSSEL-WILSON to PEYTON-MANNING via BRONCOS
PEYTON-MANNING to TOM-BRADY via RETIRED